main.go

```diff
@@ -181,6 +181,10 @@ func wrapMain() error {
 
 	secrets := map[string]interface{}{}
 	for k, v := range vargs.Secrets {
+		if v == nil {
```

Comment:
It depends on what you expect to be in that secrets map. If it's possible that a secret could be a number, you'll need to use `interface{}` and have a type check in there. If it's guaranteed all secrets will be strings, `map[string]string` should be fine.